src/screens/docs/components/victory-cursor-container/ecology.md

```diff
@@ -0,0 +1,177 @@
+# VictoryCursorContainer
+
+`VictoryCursorContainer` adds a cursor to a chart to inspect coordinates.
+The cursor can either be a 2-dimensional crosshair, or a 1-dimensional line.
+The cursor moves with the mouse (or on touch on mobile devices) along the visible domain of the chart.
+The cursor can also display a label for the active coordinates using the `cursorLabel` prop.
+
+`VictorySelectionContainer` may be used with any Victory component that works with an x-y coordinate
+system, and should be added as the `containerComponent` of the top-level component.
+However, the component that uses it must be standalone
+(`standalone={true}`), which is the default for all Victory components.
+
+Note that the cursor allows you to inspect the entire domain, not just the data points.
+If you would like to instead highlight only the data points, consider using [VictoryVoronoiContainer].
+
+```jsx
```

Comment:
oh nice! sounds good. how about if i save the playground for my next task, which is add playgrounds to all the components - does that work?